spec/requests/home_page_controller_spec.rb

```diff
@@ -43,6 +48,20 @@ def enabled?
           DiscoursePluginRegistry.reset!
         end
       end
+
+      it "should display the site description on the homepage" do
+        get "/", headers: { "HTTP_USER_AGENT" => "Googlebot" }
+
+        expect(response.status).to eq(200)
+        expect(response.body).to include("This is a test description")
+      end
+
+      it "should not display the site description on another route" do
+        get "/top", headers: { "HTTP_USER_AGENT" => "Googlebot" }
+
+        expect(response.status).to eq(200)
+        expect(response.body).to include("This is a test description")
```

Comment:
Description says "should not" but test says it should include it?

Reply:
Ah, great catch!